doc/_static/custom_scripts.js

```diff
@@ -175,6 +175,42 @@
         document.getElementById('quickstart-tour-start').onclick = startTour;
         return driver;
     };
+    
+    // Handle version selector
+    function setupVersionSelector() {
+        var selectOptions = document.querySelectorAll("#select-versions option");
+        var currentPage = window.location.pathname.split("/").slice(-1)[0];
+        var selection = [...selectOptions].filter(function(elem){
+            return elem.value === currentPage;
+        });
+
+        if(selection.length) {
+            selection.forEach(function(elem){
+                elem.setAttribute("selected", true);
+            });
+        }
+
+        // Based on https://stackoverflow.com/a/37796085
+        var isSafari = navigator.userAgent.indexOf("Safari") != -1;
+        var isChrome = navigator.userAgent.indexOf("Chrome") != -1;
+        if (isChrome || isSafari){
```

Comment:
As I'm sure you're aware, manually probing the user agent of the browser like this and changing behavior accordingly is a strongly frowned upon practice these days, in favor of checking for support of the specific features directly. It might not work on less common browsers, or if the user agent string is changed, or in many other scenarios, and seems quite fragile. Are you sure there isn't a better way of doing this?

Reply:
@dalthviz

Reply:
Will check

Reply:
Looks like this was resolved, thanks @dalthviz